backend/classes/Player.js

```diff
@@ -0,0 +1,20 @@
+class Player{
+    constructor(id){
+        this.id = id;
+
+        //what role he play in the game - Murderer, Doctor, Detective or null if not yet decided
+        this.role = null;
+
+        //what is he currently doing - playing, waiting, outside of the game etc..
+        this.activity = null;
```

Comment:
are you sure we need this? try to add variables only when you need them

Reply:
I think we do because we want to seperate the players who currently play from the players who got murderer\voted out and etc..

Reply:
so it sounds more like `state/alive/status` enum or something like that

Reply:
yes, i added activity enum. but it a little bit improvised.
for now (the very start) i think its ok